packages/zipkin/index.d.ts

```diff
@@ -85,9 +112,48 @@ declare namespace zipkin {
     }
   }
 
+  namespace model {
+    interface Endpoint {
+      constructor(args: { serviceName?: string, ipv4?: InetAddress, port?: number });
+
+      setServiceName(serviceName: string): void;
+      setIpv4(ipv4: InetAddress): void;
+      setPort(port: number): void;
+
+      isEmpty(): void;
+    }
+
+    interface Span {
+      id:             string;
+      traceId:        option.IOption<string>;
+      spanId:         string;
+      name:           string;
+      kind:           string;
+      timestamp:      number;
+      duration:       number;
+      localEndpoint:  Endpoint;
+      remoteEndpoint: Endpoint;
+      annotations:    any[];
```

Comment:
we could consider exporting model.Annotation I guess..

Reply:
Makes sense. Will do.